RecoPixelVertexing/PixelTriplets/plugins/CACell.h

```diff
@@ -0,0 +1,233 @@
+#ifndef CACELL_H_
+#define CACELL_H_
```

Comment:
please match the include guard to the package/file name 
```RecoPixelVertexing_PixelTriplets_CACell_h ``` (CACELL is too short)